lib/index.ts

```diff
@@ -726,8 +720,8 @@ export default class Backburner {
   private _ensureInstance(): DeferredActionQueues {
     let currentInstance = this.currentInstance;
     if (currentInstance === null) {
-      const next = this._platform.next || this._platform.setTimeout; // TODO: remove the fallback
```

Comment:
`next` is already used without fallback on line 148, so probably can be removed here too